### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by thinkh**
**prerequisites**: 
 * [x] branch is up-to-date with the branch to be merged with, i.e. develop
 * [x] build is successful
 * [x] code is cleaned up and formatted 

### Summary

* Use lighter background color on hover over column header
* Rename `Mapping / Scaling Type` to `Scaling` in mapping dialog
* Sort visualizations in ChangeRendererDialog by label
* Increase spacing for change renderer groups
* Use slightly darker primary action in dialogs
* Increase bottom space for histogram in side panel 
* Change placeholder text for search input
* Decrease font-size for side panel header
* Center row renderer content
* Modify checkbox label text for missing values
* Modify style for selected items indicator in side panel
* Modify style of side panel collapse toggle button
* Center side panel header text
* Rename `Stratify` to `Group by`

### CODE REVIEW

**Comment by sgratzl** [comment]
starting with the upcoming release there is a new prerequisite for PR: 

* [ ] tested with Firefox 52, Firefox 57+, Chrome 64+, MS Edge 16+

**Comment by sgratzl** [comment]
notes: 

image

image


new side collapser is overlapping with the add column chooser

image

why so much space after the checkbox?

> Change placeholder text for search input

but it is not a search operation (different action) but actually filtering the data, thus is the label misleading right now. 

> Center row renderer content

do you know how "expensive" in terms of layout styling this is?

**Comment by thinkh** [comment]
>  tested with Firefox 52, Firefox 57+, Chrome 64+, MS Edge 16+

Which tool do you use to test all these browsers? I don't have all these browsers installed on my machine...

**Comment by thinkh** [comment]
I addressed your styling comments in the last commits. I moved most of the styles then to the lineup_app (branch taggle_user_study).

> do you know how "expensive" in terms of layout styling this is?
No, I'm not aware of it. Can you please explain/show it?

File: src/renderer/StringCellRenderer.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import {IDataRow, IGroup} from '../model';
+import { IDataRow, IGroup } from '../model';
```
Comment (sgratzl) [inline src/renderer/StringCellRenderer.ts]: why the formatting, which should even be disabled with the included `.vscode` settings?

Comment (thinkh) [reply src/renderer/StringCellRenderer.ts]: You are right. The problem was that I did not open the lineupjs directory as VS Code root, but the parent directory. There were no `.vscode` settings available. I fixed the import with the last commit.

**Pull request merged**